### PR TITLE
86 create org page

### PR DESCRIPTION
**Pull request by fadeomar**
#### Issues this PR relates to
#86 
**NOTE:
there is an issue in the back and I opened a new one for it #124, it should be solved before merging this work I've temporary commented the code that caused the issue but didn't push it with this PR 
=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. 

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: server/modules/user/controllers/index.js [hunk 1]
```diff
@@ -30,20 +32,14 @@ router.get(
...
-  '/',
-  csrfProtection,
-  authenticate(),
-  authorize([
-    /* super admin */
-  ]),
-  updateUser,
-);
+router.get('/admin-users', getAdminUsers);
+router.patch('/update-user-role', csrfProtection, authenticate(), updateUser);
```
Comment (RamyAlshurafa) [inline server/modules/user/controllers/index.js:36]: add authentication and authorization for both routes

File: server/modules/user/controllers/index.js [hunk 2]
```diff
@@ -30,20 +32,14 @@ router.get(
...
 router.post('/signup', csrfProtection, signup);
 router.post('/login', csrfProtection, login);
 router.post('/logout', csrfProtection, logout);
 router.post('/reset-password-link', csrfProtection, resetPasswordLink);
 router.post('/update-password', csrfProtection, updatePassword);
-router.post('/reset-password-link', csrfProtection, resetPasswordLink);
+
+router.delete('/delete-user', deleteUser);
```
Comment (RamyAlshurafa) [inline server/modules/user/controllers/index.js:43]: same here, add auth middlewares

File: client/src/pages/Organisations/index.js [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+        </Col>
+        <Col w={[0, 0, 4]}>
+          <T.P color="neutralMain" mb="6" weight="bold">
+            Role
+          </T.P>
+        </Col>
+      </Row>
+      {users &&
+        users.map((user) => (
+          <Row>
```
Comment (RamyAlshurafa) [inline client/src/pages/Organisations/index.js:157]: KEY

**Review by RamyAlshurafa — CHANGES_REQUESTED** [review]
https://www.loom.com/share/60a16efee4a14a68a9be32e9a2908688
- [x] hide clear icon on select component
- [ ] after cancelling it should reset the selected value to the prev value
- [ ] editing roles should be allowed for super admin only
- [ ] add on delete cascade on any db relation with users table

File: client/src/App.js [hunk 4]
```diff
@@ -97,6 +97,14 @@ function App() {
                     isPrivate
                     allowedRoles={[userRoles.SUPER_ADMIN]}
                   />
+                  <Route
+                    exact
+                    // path={navRoutes.ADMIN.DASHBOARD}
+                    path="/test"
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply client/src/App.js:103]: done

File: client/src/api-calls/users.js [hunk 5]
```diff
@@ -13,6 +13,40 @@ const getUserById = async ({ id, options }) => {
   }
 };
 
+const deleteUser = async ({ id }, { options } = {}) => {
+  try {
+    const data = await axios.delete(`${USERS_BASE}/delete-user`, {
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply client/src/api-calls/users.js:18]: done

File: client/src/api-calls/users.js [hunk 6]
```diff
@@ -13,6 +13,40 @@ const getUserById = async ({ id, options }) => {
...
+  } catch (error) {
+    const err = handleError(error, options);
+    return { error: err };
+  }
+};
+
+const updateUserRole = async (updates, { options } = {}) => {
+  try {
+    const { data } = await axios.patch(
+      `${USERS_BASE}/update-user-role`,
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply client/src/api-calls/users.js:31]: done

File: client/src/api-calls/users.js [hunk 7]
```diff
@@ -13,6 +13,40 @@ const getUserById = async ({ id, options }) => {
...
+    );
+    return { data };
+  } catch (error) {
+    const err = handleError(error, options);
+    return { error: err };
+  }
+};
+const getAdminUsers = async () => {
+  try {
+    const { data } = await axios.get(`${USERS_BASE}/admin-users`);
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply client/src/api-calls/users.js:42]: done

File: server/modules/user/controllers/index.js (hunk 1 above)
Comment (fadeomar) [reply server/modules/user/controllers/index.js:36]: done

File: server/modules/user/controllers/index.js (hunk 2 above)
Comment (fadeomar) [reply server/modules/user/controllers/index.js:43]: done

File: client/src/pages/Organisations/index.js (hunk 3 above)
Comment (fadeomar) [reply client/src/pages/Organisations/index.js:157]: done

**Review by RamyAlshurafa — APPROVED** [review]

**Pull request merged**